pkg/controller/schedule_controller_test.go

```diff
@@ -97,23 +97,23 @@ func TestProcessSchedule(t *testing.T) {
 			fakeClockTime:        "2017-01-01 12:00:00",
 			expectedErr:          false,
 			expectedPhase:        string(velerov1api.SchedulePhaseEnabled),
-			expectedBackupCreate: builder.ForBackup("ns", "name-20170101120000").ObjectMeta(builder.WithLabels(velerov1api.ScheduleNameLabel, "name")).NoTypeMeta().Result(),
```

Comment:
What's the main reason for the Backup to not have any TypeMeta, is it not needed because the client fills it in automatically when creating the backup?

Reply:
I believe that's that case, yes.